### PR TITLE
Intro conclusion customizing

### PR DESCRIPTION
**Pull request by drbalar**
added fields to the backend scenario to allow customizing of intro and conclusion text.

Intro:
- header text

Conclusion:
- header text
- paragraph text
- share link

All fields are optional.

UI did not really change so won't demo this yet.

### CODE REVIEW

File: app/library/api_schemas.py [hunk 1]
```diff
@@ -45,10 +45,20 @@
         "publish_link": {"type": "string", "pattern": r"^[\S]{1,50}$"},
         "preview_link": {"type": "string", "pattern": r"^[\S]{1,50}$"},
         "expected_solve_time": {"type": "string", "pattern": r"^[a-zA-Z0-9 _-]{,50}$"},
+        "introduction_data": {
+            "type": "object",
+            "properties": {
+                "header_text": {"type": "string"},
```
Comment (ahmedhamodi) [inline app/library/api_schemas.py:51]: Do we also want a paragraph text for intro? Or just conclusion?

File: app/routes/ui/landing_page.py [hunk 2]
```diff
@@ -31,9 +31,11 @@ async def get(self, scenario_friendly_name):
                 "landing_page.html",
                 scenario_name=scenario_obj.name,
                 scenario_friendly_name=scenario_obj.friendly_name,
+                scenario_introduction_data=scenario_obj.introduction_data,
             )
         except ValueError as e:
             self.write_error(status_code=404, message=str(e))
-        except Exception:
+        except Exception as e:
+            print(e)
```
Comment (ahmedhamodi) [inline app/routes/ui/landing_page.py:39]: Do we want this print still?

**Review by ahmedhamodi — COMMENTED** [review]
Some thoughts, but overall code lgtm

File: app/routes/ui/landing_page.py (hunk 2 above)
Comment (drbalar) [reply app/routes/ui/landing_page.py:39]: Yikes, no

File: app/library/api_schemas.py (hunk 1 above)
Comment (drbalar) [reply app/library/api_schemas.py:51]: I imagined the intro being just a title and a play button. What do you think?

Comment (ahmedhamodi) [reply app/library/api_schemas.py:51]: I'm good with this for now, but we may want to add in the future.

File: app/models/scenario.py [hunk 3]
```diff
@@ -16,7 +16,18 @@ class Scenario(BaseModel):
     publish_link = Column(Text, nullable=True)
     preview_link = Column(Text, nullable=True)
     expected_solve_time = Column(Text, nullable=False, default="")
-    conclusion_data = Column(JSONB, nullable=False, default={"share_link": ""})
+    introduction_data = Column(
+        JSONB, nullable=False, default={"header_text": "Welcome to the escape room."}
```
Comment (jaydhulia) [inline app/models/scenario.py:20]: Let's make this configurable, similar to how we do transitions, so we can change it easily once DCC gives us the default ones they want. So like:
```
default=config.get("default_data.introduction_data")
```

File: app/models/scenario.py [hunk 4]
```diff
@@ -16,7 +16,18 @@ class Scenario(BaseModel):
...
+    )
+    conclusion_data = Column(
+        JSONB,
+        nullable=False,
+        default={
```
Comment (jaydhulia) [inline app/models/scenario.py:25]: Let's make this configurable, similar to how we do transitions, so we can change it easily once DCC gives us the default ones they want. So like:
```
default=config.get("default_data.conclusion_data")
```

**Review by jaydhulia — APPROVED** [review]
Couple of minor comments, LGTM otherwise! Approved!

**Review by jaydhulia — APPROVED** [review]

**Pull request merged**